wcomponents-theme/src/main/js/wc/file/util.js

```diff
@@ -0,0 +1,97 @@
+define(["wc/dom/uid", "wc/file/getMimeType"], function(uid, getMimeType) {
+	"use strict";
+	var instance = new FileUtil();
+	/**
+	 * Used when checking the newly created image is named with the correct extension.
+	 * If it does not already match any in the array then the extension at index zero will be appended.
+	 **/
+	FileUtil.prototype.mimeToExt = {
+		"image/jpeg": ["jpeg", "jpg"],
+		"image/png": ["png"],
+		"image/webp": ["webp"]
+	};
+
+	function FileUtil() {
+		/**
+		 * Converts a generic binary blob to a File blob.
+		 * @param {Blob} The binary blob.
```

Comment:
Where is the parameter name?
Here you have defined a parameter called `The` which has the type `{Blob}`.

Reply:
Good catch - replaced on JSDoc error with another. JSDoc is *not* JavaDoc.